src/http_client/mod.rs

```diff
@@ -63,6 +95,23 @@ impl Client {
         })
     }
 
+    /// Checks whether the [`Client`] is pointing to a valid Etebase server.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use etebase::Client;
+    ///
+    /// let invalid_client = Client::new("ExampleClient", "https://example.com").unwrap();
+    /// assert!(!invalid_client.has_valid_server().unwrap());
+    ///
+    /// let valid_client = Client::new("ExampleClient", "https://api.etebase.com/").unwrap();
+    /// assert!(valid_client.has_valid_server().unwrap());
+    /// ```
+    pub fn has_valid_server(&self) -> Result<bool> {
```

Comment:
Maybe is_valid_server? I liked that better.

Reply:
To me that would imply "is this object (of type `Client`) a valid server?", which is a bit of a nonsensical question. What do you think of `is_server_valid()`?

Reply:
Sounds good.

Reply:
Done!